client/src/entrypoints/admin/telepath/blocks.js

```diff
@@ -612,6 +696,36 @@ class StreamBlock {
     }
   }
 
+  setError(errorList) {
+    if (errorList.length !== 1) {
+      return;
+    }
+    const error = errorList[0];
+
+    // Non block errors
+    const container = this.container[0];
+    container.querySelectorAll(':scope > .help-block .help-critical').forEach(element => element.remove());
+
+    if (error.nonBlockErrors.length > 0) {
+      // Add a help block for each error raised
+      error.nonBlockErrors.forEach(errorText => {
+        const errorElement = document.createElement('p');
+        errorElement.classList.add('help-block');
+        errorElement.classList.add('help-critical');
+        errorElement.innerText = errorText;
+        container.insertBefore(errorElement, container.childNodes[0]);
+      });
+    }
+
+    // Block errors
+    // eslint-disable-next-line no-restricted-syntax
+    for (const blockId in error.blockErrors) {
```

Comment:
`blockId` here (and in ListBlock) would be better named `blockIndex` to avoid confusing it with the UUID `id` field of StreamChild.